### DISCUSSION TITLE
Rootless server

### DISCUSSION THREAD

**Discussion by jmodze** [lloesche/valheim-server-docker#594] [Q&A]
Is it possible to run this image as rootless user? I've seen other games don't need root access. When simply changing docker run to use --user without root access I can't start the container.

**Comment by insunaa** [comment]
You can nest containers: Run the docker container inside of an unprivileged LXC container. Works great for me

**Comment by schildbach** [comment]
You can at least use the PUID and PGID environment variables to make Valheim Server, Valheim-Updaterm, SteamCMD and a couple more run as an unprivileged user. There will be some root processes left however (supervisord, syslogd, cron).
